api/lib/infrastructure/repositories/participant-result-repository.js

```diff
@@ -21,13 +22,24 @@ const ParticipantResultRepository = {
 };
 
 async function _getParticipationResults(userId, campaignId) {
-  const { isCompleted, campaignParticipationId, sharedAt, assessmentCreatedAt, participantExternalId, masteryRate } =
-    await _getParticipationAttributes(userId, campaignId);
+  const {
+    isCompleted,
+    campaignParticipationId,
+    sharedAt,
+    assessmentCreatedAt,
+    participantExternalId,
+    masteryRate,
+    isFlash,
+    assessmentId,
+  } = await _getParticipationAttributes(userId, campaignId);
 
   const knowledgeElements = await _findTargetedKnowledgeElements(campaignId, userId, sharedAt);
 
   const acquiredBadgeIds = await _getAcquiredBadgeIds(userId, campaignParticipationId);
 
+  let estimatedFlashLevel;
+  if (isCompleted && isFlash) estimatedFlashLevel = await _getEstimatedFlashLevel(assessmentId);
```

Comment:
Le `isCompleted` est-il nécessaire ?
```suggestion
  if (isFlash) estimatedFlashLevel = await _getEstimatedFlashLevel(assessmentId);
```